### PR TITLE
currentCollector to Generator - db migration

### PR DESCRIPTION
**Pull request by pallavi2209**
db validation for collector and tests coming in next PRs

### CODE REVIEW

File: tests/api/v1/collectors/utils.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const expectedProps = [
   'aspects', 'collectors', 'connection', 'context', 'createdAt', 'createdBy',
   'deletedAt', 'description', 'generatorTemplate', 'helpEmail', 'helpUrl', 'id',
   'isActive', 'isDeleted', 'name', 'subjectQuery', 'subjects', 'tags',
-  'updatedAt', 'user',
+  'updatedAt', 'user', 'currentCollector',
```
Comment (iamigo) [inline tests/api/v1/collectors/utils.js]: let's keep the list in alphabetical order

**Comment by pallavi2209** [comment]
Tested locally - let me test in devbox as well. Api changes in separate PR, but before that I will complete db changes - need to add validations for currentCollector name as one of the assigned list of collectors of generator - plus corresponding tests.

**Comment by pallavi2209** [comment]
Tested in devbox with migratedb and undo-migratedb scripts - works as expected.

**Comment by harshkothari410** [comment]
Can you resolve this conflict @pallavi2209 ?

**Comment by pallavi2209** [comment]
@harshkothari410 resolved - can you confirm and merge then please?

**Comment by pallavi2209** [comment]
Got 3 approvals - Merging this PR to continue working on the top of these changes.

**Pull request merged**